README.md

```diff
@@ -1,10 +1,11 @@
-# Howso Engine&trade;
 
-The Howso Engine is an application that enables data exploration and machine learning capabilities through
-the creation and use of Trainees that help users store, explore, and analyze the relationships in their data.
-The Howso Engine utilizes a instance-based learning approach with strong ties to the [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) and [information theory](https://en.wikipedia.org/wiki/Information_theory).
+<div align="left">
+  <img src="https://docs.howso.com/_static/howso_logo_icon.svg">
+</div>
 
-Trainees are designed to be quite versatile, being a single model that after training a dataset can do all of the following without the need to retrain:
+The Howso Engine&trade; is a natively and fully explainable ML engine, serving as an alternative to black box AI neural networks. Its core features give users data exploration and machine learning capabilities through the creation and use of Trainees that help users store, explore, and analyze the relationships in their data. Howso leverages an instance-based learning approach with strong ties to the [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) and [information theory](https://en.wikipedia.org/wiki/Information_theory) to scale for real world applications.
+
+At the core of Howso is the concept of a Trainee, a collection of cases that comprise knowledge. In traditional ML, this is typically referred to as a model, but a Trainee may additionally include metadata, parameters, details of feature attributes, with data lineage and provenance. Unlike traditional ML, Trainees are designed to be versatile, a single model that after training a dataset can do the following without the need to retrain:
```

Comment:
Slightly confusing sentence: "... may additionally include metadata, parameters, details of feature attributes, with data lineage and provenance."

Does "lineage and provenance" apply to all the things in this list? Or just "details of feature attributes?" In any case, I'd say maybe rework this sentence to make it more clear.